trunk/includes/pardot-api-class.php

```diff
@@ -428,11 +428,7 @@ function get_response( $item_type, $args = array(), $property = 'result', $paged
 			)
 		);
 
-		$headers = [
-			'Authorization' => 'Pardot user_key=' . $this->user_key . ',api_key=' . $this->api_key
-		];
-
-		$http_response = wp_remote_request(
+		$http_response = wp_remote_post(
 			$this->_get_url( $item_type, $args ),
```

Comment:
My only concern here is that the authorization keys are in $args, which is in both the URL and the body. Depending on how the enforcement is implemented, if the query string is checked before the request type, this could cause requests to fail. A cursory look at the implementation doesn't say one way or another.

Reply:
@adelawalla `_get_url()` is a misleading function name (that needs to be updated). This still shouldn't pass the params in the URL AFAIK.